README.rst

```diff
@@ -14,9 +14,9 @@ The **Variantstore** is a Java/Groovy-based service application implemented usin
 
 Features
 --------
-- Import metadata (JSON files using this `schema <https://github.com/qbicsoftware/mtb-metadata-specs/blob/master/schemes/mtb/variants.metadata.schema.json>`_)
 - Import variants (VCF files, annotated using `SnpEff <http://snpeff.sourceforge.net>`_ or `VEP <https://www.ensembl.org/info/docs/tools/vep/index.html>`_)
-- Import gene information (Ensembl, GFF3 files)  
+- Import metadata in JSON together with variants
```

Comment:
Did the metadata variant schema change? If not i'd include the previous linkage to the schema here

Reply:
Or even better link the usage section here 👍